pyartcd/pyartcd/pipelines/gen_assembly.py

```diff
@@ -0,0 +1,71 @@
+import click
+from doozerlib import exectools
+from pyartcd import exceptions, github
+
+
+@click.command()
+@click.option('--nightlies', required=True)
+@click.option('--custom', required=False)
+@click.option('--in_flight_prev', required=True)
+@click.option('--previous', required=False)
+@click.option('--build_version', required=True)
+@click.option('--assembly_name', required=True)
+def main(nightlies, custom, in_flight_prev, previous, build_version, assembly_name):
+    # Create doozer cmd arguments
+    nightly_args = ""
+    for nightly in nightlies.split(","):
+        nightly_args += f" --nightly {nightly.strip()}"
+
+    cmd = f"--group openshift-{build_version} release:gen-assembly --name {assembly_name} from-releases {nightly_args}"
+    if custom:
+        cmd += ' --custom'
+        if (in_flight_prev and in_flight_prev != 'none') or previous:
```

Comment:
Let's keep the != none check case-insensitive

Reply:
Done!